### PR TITLE
ref(frame-line): Convert test to RTL and TS

### PR DESCRIPTION
**Pull request by priscilawebdev**

### CODE REVIEW

File: tests/js/sentry-test/reactTestingLibrary.tsx [hunk 1]
```diff
@@ -50,4 +50,4 @@ const mountWithTheme = (ui: React.ReactElement, options?: ContextRenderOptions)
 };
 
 export * from '@testing-library/react';
```
Comment (scttcper) [inline tests/js/sentry-test/reactTestingLibrary.tsx:52]: shouldn't within have already been exported from here

**Review by scttcper — APPROVED** [review]
other than the one comment lgtm

File: tests/js/spec/components/events/interfaces/frame/line.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          isExpanded
+        />
+      );
+
+      for (const [key, value] of Object.entries(vars)) {
+        const row = getByText(key).closest('tr');
+        expect(row).toBeTruthy();
+
+        if (!row) {
+          return;
```
Comment (scttcper) [inline tests/js/spec/components/events/interfaces/frame/line.spec.tsx:143]: return or continue here?

File: tests/js/sentry-test/reactTestingLibrary.tsx (hunk 1 above)
Comment (priscilawebdev) [reply tests/js/sentry-test/reactTestingLibrary.tsx:52]: yes... good catch

File: tests/js/spec/components/events/interfaces/frame/line.spec.tsx (hunk 2 above)
Comment (priscilawebdev) [reply tests/js/spec/components/events/interfaces/frame/line.spec.tsx:143]: return... row can not be `null` as a `within` func parameter

**Pull request merged**